### ISSUE TITLE
Помогите, пожалуйста,  с биндом кнопоу через команду джарвису!

### ISSUE THREAD

**Issue by DemisXx** [Priler/jarvis#112]
В общем тема такая, я научил его открывать закрывать ютуб хром и прочее, так вот хотел бы, чтобы при открытии видео он раскрывал его на весь экран голосовой командой, в ютубе это клавиша "f" как написать код или сделать бинд, чтобы джарвис при "на весь экран" или "видео на весь экран" нажимал клавишу "f"? За ранее благодарю!

**Comment by DemisXx** [comment]
Короче решил проблему, через .bat файлы через консоль, а точнее nircmd

**Comment by ReekyLeon** [comment]
Да, конечно позновато, но всё же напишу, может кому понадобиться)
Через AutoHotKey можно спокойно забиндить кнопку "f", всё просто, надо прописать лишь пару строчек:

**Send F** (нажимает кнопку "F")
**Sleep, 100** (задержка в виде 100 милисекунд)
**Send А** (нажимает ту же кнопку "F" но при условии если раскладка стоит русская, тут указа русская буква "А")

Лично я пользовался именно таким способом)
